apps/docs/stories/components/PostalCodeForm/Docs.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Title, ArgTypes, Source } from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExampleFull } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiPostalCodeForm from './index.stories';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Postal Code Form"
+  Components="justifi-postal-code-form"
+  of={JustifiPostalCodeForm}
+/>
+
+<Title />
+
+---
+
+Renders a form for collecting the postal code as part of a checkout flow.
+
+This component is **designed to be used within** the `justifi-modular-checkout` and **does not accept props directly**. Instead, it relies on shared state passed through the Stencil Store, managed by the wrapper component.
+
+> **Note:** This component is specifically designed to be used in conjunction with the `justifi-card-form` component. Alternatively, you can provide the postal code by passing a `billingInfo` object with an `address_postal_code` field as an argument to the `submitCheckout` method called on the wrapper.
```

Comment:
Change to 'passing a `billingInformation` object with an `address_postal_code` field as an argument to the `submitCheckout` method called on the `justifi-modular-checkout` component. See the Modular Checkout docs for more information.'

Please make Modular Checkout docs a link to that page